book/website/community-handbook/presenting.md

```diff
@@ -0,0 +1,14 @@
+# Presenting About the Turing Way
+Talks are a great way to engage new communities, teach people about open and reproducible practices, and expand the reach of the Turing Way by introducing the book and inviting new contributors. We encourage any Turing Way contributor to present on behalf of the Turing Way, and we ask that you use the guidelines detailed below to shape your talk and ensure consistency with TTW style. 
+
```

Comment:
Could we add in her a section on: 
### Suggesting venues for talks
If you would like _The Turing Way_ to talk at your event or at a community meeting, please do get in touch with our Community Manager or promote the opportunity on our #community Slack channel.